spark-bigquery-dsv2/spark-3.1-bigquery/src/test/java/com/google/cloud/spark/bigquery/integration/Spark31DirectWriteIntegrationTest.java

```diff
@@ -28,4 +29,8 @@ public Spark31DirectWriteIntegrationTest() {
 
   // tests from superclass
 
+  @Test
+  public void testFoo() throws Exception {
```

Comment:
Seems like this was added by mistake? I see similar tests in Spark31ReadByFormatIntegrationTest and Spark31ReadIntegrationTest as well

Reply:
oops :-)